aspose_barcode_cloud/models/file_version.py

```diff
@@ -44,171 +44,21 @@ class FileVersion(object):
       attribute_map (dict): The key is attribute name
                             and the value is json key in definition.
     """
-    swagger_types = {
-        "name": "str",
-        "is_folder": "bool",
-        "modified_date": "datetime",
-        "size": "int",
-        "path": "str",
-        "version_id": "str",
-        "is_latest": "bool",
-    }
-
-    attribute_map = {
-        "name": "Name",
-        "is_folder": "IsFolder",
-        "modified_date": "ModifiedDate",
-        "size": "Size",
-        "path": "Path",
-        "version_id": "VersionId",
-        "is_latest": "IsLatest",
-    }
-
-    def __init__(
-        self, name=None, is_folder=None, modified_date=None, size=None, path=None, version_id=None, is_latest=None
-    ):  # noqa: E501
+    swagger_types = {"version_id": "str", "is_latest": "bool"}
+
+    attribute_map = {"version_id": "VersionId", "is_latest": "IsLatest"}
```

Comment:
It would be nice to cover this changes with tests

Reply:
Yes it would be.